include/rom4.h

```diff
@@ -0,0 +1,15 @@
+//
+// Created by scott on 9/15/2017.
+//
+
+#ifndef POKEEMERALD_ROM4_H
```

Comment:
Please be consistent and use `GUARD_ROM4_H`. Also, we've renamed rom4 to overworld in pokeruby.